database/migrations/2024_05_02_020045_create_task_groups_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('task_groups', function (Blueprint $table) {
+            $table->id();
+            $table->string('title', 100);
```

Comment:
chỗ này tại sao phải limit nó là 100 kí tự em nhỉ

Reply:
Dạ, vì nếu quá nhiều ký tự thì bên FE cũng không đủ chỗ hiển thị đủ nội dung. Nên em chỉ cho 100 ký tự thôi ạ.

Reply:
Anh nghĩ đó không phải là vấn đề của bên BE, nếu bên FE ở ngoài màn hình list không có đủ chỗ để hiển thị nội dung thì có thể sử dụng **...** , vào detail của công việc sẽ show đầy đủ